### PR TITLE
Bugfix: Fixing HTML proof errors

### PR DESCRIPTION
**Pull request by yuvenus**

### CODE REVIEW

**Review by ChrisWeissmann — APPROVED** [review]

File: _layouts/default.html [hunk 1]
```diff
@@ -11,7 +11,6 @@
     <div class="site-container">
       {% include top-nav-bar.html %}
       <div class="content-container">
-        {% include hero.html %}
```
Comment (langiole) [inline _layouts/default.html:14]: 😢

Comment (langiole) [reply _layouts/default.html:14]: @yuvenus was there a discussion that led to this being removed?

Comment (yuvenus) [reply _layouts/default.html:14]: unfortunately it was one of the things the HTML proofer was not happy about, nor did it have any immediate functionality, hence its removal :( however, if it's actually needed and the proofer needs to be wrangled, let me know and we can add it back in 👍

Comment (yuvenus) [reply _layouts/default.html:14]: @langiole https://github.com/citrineos/citrineos.github.io/pull/34 😄

**Pull request merged**